src/main/scala/cromwell/webservice/ApiDataModels.scala

```diff
@@ -7,54 +7,67 @@ import cromwell.engine.backend.StdoutStderr
 
 import scala.annotation.meta.field
 
+@ApiModel(value = "WorkflowValidate")
+case class WorkflowValidateResponse
+(
+  @(ApiModelProperty@field)(required = true, value = "The validation of the workflow")
```

Comment:
This is a combo of stuff:

`@ApiModelProperty` is straight from the dark heart of [`swagger-core`](https://github.com/swagger-api/swagger-core/blob/v1.3.12/modules/swagger-annotations/src/main/java/com/wordnik/swagger/annotations/ApiModelProperty.java#L27).

`@field` is the extra special scala [magic](http://www.scala-lang.org/api/2.11.7/index.html#scala.annotation.meta.package).

The lack of spaces is the nice touch of IntelliJ IDEA's scala plugin + default auto formatting.